models/tridimensional/data_kleinstiver/cas9_mutants_stats.py

```diff
@@ -137,3 +137,106 @@ def main():
 
 if __name__ == '__main__':
     main()
+
+    
+def idx_freq(pam):
+    '''
+    Given a pam site returns the indices of mutation and frequency of mutation
+    occurring at said indices
+    '''
+    
+    mut_lst = []
+    for mutant in mutants_kleinstiver:
+        if mutant['pam'] == pam:
+            mut_lst.append(mutant)
```

Comment:
`mut_lst = [mutant for mutant in mutants_kleinstiver if (mutant['pam'] == pam)]`

this abbreviates these 4 lines and is faster